### DISCUSSION TITLE
Create `solid-jotai-x` library

### DISCUSSION THREAD

**Discussion by rtritto** [udecode/jotai-x#15] [Ideas]
Create `solid-jotai-x` library for `solid-jotai` to use Jotai with SolidJS.

Related: [solid-jotai](https://github.com/wobsoriano/solid-jotai)

**Comment by rtritto** [comment]
FYI @12joan @zbeyens @wobsoriano

**Comment by zbeyens** [comment]
Contributions are welcome.

**Reply by rtritto** [reply]
I created #16
